Hudl.Mjolnir/Config/MjolnirConfiguration.cs

```diff
@@ -100,11 +102,11 @@ public CommandConfiguration GetCommandConfiguration(string groupKey = null)
         /// <param name="key">Bulkhead configuration for a given key. Default value returned if non-existent or 
         /// null.</param>
         /// <returns></returns>
-        public BulkheadConfiguration GetBulkheadConfiguration(string key = null)
+        public BulkheadConfiguration GetBulkheadConfiguration(GroupKey key)
```

Comment:
Are the other methods (`GetCommandConfiguration`, `GetBreakerConfiguration`) switching over to `GroupKey` instead of `string` as well?